server/fleet/errors.go

```diff
@@ -479,3 +479,14 @@ func Cause(err error) error {
 		err = uerr
 	}
 }
+
+// OrbitError is used for orbit endpoints, to return an error message along
+// with a failed request's response.
+type OrbitError struct {
```

Comment:
This is not a new type, it's just the existing `service.orbitError` type that I moved to the `fleet` package and exported to reuse in the `ee/server/service` package.